kms/tpmkms/tpmkms_simulator_test.go

```diff
@@ -433,8 +442,62 @@ func TestTPMKMS_CreateKey(t *testing.T) {
 	}
 }
 
+func TestTPMKMS_CreateKey_Capabilities(t *testing.T) {
+	tpmWithNoCaps := newSimulatedTPM(t, &tpmp.Capabilities{}, withAK("ak1"))
```

Comment:
The `withAK` option doesn't seem to be used in the test, as it's creating keys; not attesting their creation.